resolwe_bio/descriptors/reads_template.yml

```diff
@@ -0,0 +1,13 @@
+- name: reads_annotation
```

Comment:
Rename the file to reads.yml. The file is in the descriptors folder so it is obvious it contains a template/descriptor schema.